src/schemathesis/specs/openapi/schemas.py

```diff
@@ -414,6 +417,9 @@ def make_operation(
         self.dispatch_hook("before_init_operation", HookContext(operation=operation), operation)
         return operation
 
+    def _parse_responses(self, definition: dict[str, Any], scope: str) -> Responses:
```

Comment:
It will be nice to pass a spec-specific `adapter` and offload all implementation to it instead of doing inheritance stuff.